### PR TITLE
✨ update runtimes

### PR DESCRIPTION
**Pull request by ctcpip**
resolves https://github.com/tc39/tg3/issues/10

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -22,9 +22,11 @@ Alternately, send an email to `security@tc39.es`
...
-| JavaScriptCore  | Safari                 | [Report](https://webkit.org/security-policy/)   |
-| Node            |                        | [Report](https://nodejs.dev/en/about/security/) |
-| SpiderMonkey    | Firefox                | [Report](https://www.mozilla.org/security/)     |
-| V8              | Chrome, Chromium, Edge | [Report](https://v8.dev/docs/security-bugs)     |
+| Engine/Runtime | Used In                            | Link to Report                                             |
+| -------------- | ---------------------------------- | ---------------------------------------------------------- |
+| JavaScriptCore | Safari, Bun                        | [Report](https://webkit.org/security-policy/)              |
+| SpiderMonkey   | Firefox                            | [Report](https://www.mozilla.org/security/)                |
+| V8             | Chrome, Chromium, Edge, Node, Deno | [Report](https://v8.dev/docs/security-bugs)                |
+| Node           |                                    | [Report](https://nodejs.dev/en/about/security/)            |
```
Comment (ljharb) [inline SECURITY.md:30]:
```suggestion
| Chakra         | Edge < 80                       | [Report](https://github.com/chakra-core/ChakraCore?tab=readme-ov-file#security) |
| Node           |                                    | [Report](https://nodejs.dev/en/about/security/)            |
```

**Review by ljharb — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (ctcpip) [reply SECURITY.md:30]: security fixes only till 2021 though

Comment (ctcpip) [reply SECURITY.md:30]: last release was in 2020

Comment (ljharb) [reply SECURITY.md:30]: that's just from microsoft; the community continues to maintain and support it.

**Pull request merged**